slam-ws/src/caffeine/launch/caffeine_gazebo.launch

```diff
@@ -21,7 +21,7 @@
     <param name="robot_description" command="xacro --inorder '$(arg model)'"/>
     
     <node name="urdf_spawner" pkg="gazebo_ros" type="spawn_model"
-        args="-z 1.0 -unpause -urdf -model robot -param robot_description" respawn="false" output="screen"/>
+        args="-y -21.5 -z 1.0 -unpause -urdf -model robot -param robot_description" respawn="false" output="screen"/>
```

Comment:
Great!

Nit: I think it would be better if we passed these values as command line params. I say this because during testing, it might be useful to be able to start the robot anywhere.

We can do something like 
```
<arg name="spawn_y" />
<arg name="spawn_z" />
```

and then
```
args = "-y $(arg spawn_y) -z $(arg spawn_z) -unpause ...
```
When running, we can then do
```
roslaunch caffeine_gazebo.launch spawn_y:=-21.5 spawn_z:=1.0 
```

Thoughts?

Reply:
seems reasonable, could be helpful for testing more quickly at certain parts of the course -- implemented

Reply:
I agree it would be nice to be able to choose the spawn location on the horizontal plane (y-z plane?). Likewise, I think we should add a yaw value as well. But by default it should use it's current values (tho I want it to look 180 degrees the other direction)